lib/cc/service/http.rb

```diff
@@ -99,4 +105,9 @@ def formatted_post_response(response, url, body)
       message: "Success",
     }
   end
+
+  def skip_webhook_validation?
+    var = ENV["CODECLIMATE_SKIP_WEBHOOK_VALIDATION"] || ""
```

Comment:
Would it make more sense to call this something like `ALLOW_INTERNAL_WEBHOOKS`? "Validation" seems ambiguous.